packages/terra-form-select/src/SelectOption.scss

```diff
@@ -0,0 +1,38 @@
+:local {
+  .option {
```

Comment:
These options have a bulleted disc, is this intentional?

![image](https://user-images.githubusercontent.com/6720991/32243028-51cd5f56-be42-11e7-821f-bdf000c3c188.png)